### ISSUE TITLE
pylint reports false positive error for lru_cache with maxsize omitted

### ISSUE THREAD

**Issue by mdegat01** [PyCQA/pylint#6180]
### Bug description

Starting in `v2.13.0` pylint reports warning if you decorate a method with `lru_cache` but do not provide a value for `maxsize` (#5670) . It does this even if you omit a value for `maxsize` and just use `@lru_cache` or `@lru_cache()`.

This is incorrect, the `lru_cache` decorator [defaults maxsize to 128 if omitted](https://github.com/python/cpython/blob/main/Lib/functools.py#L479) so it should not warn about this. It should only warn if `maxsize` is explicitly set to `None` by doing `@lru_cache(maxsize=None)` as that is what can create memory leaks.

### Command used

```shell
pylint supervisor/host/manager.py
```

### Pylint output

```shell
************* Module supervisor.host.manager
supervisor/host/manager.py:71:5: W1516: 'lru_cache' without 'maxsize' will keep all method args alive indefinitely, including 'self' (lru-cache-decorating-method)
```

### Expected behavior

No warning since we are using the default maxsize of 128.

### Pylint version

```shell
pylint 2.13.4
```

### OS / Environment

macOS 12.2.1
Python 3.10.2
pip 22.0.4
